blueocean-dashboard/src/main/js/components/karaoke/components/Step.jsx

```diff
@@ -77,11 +77,6 @@ export class Step extends Component {
     }
 
     render() {
-        // early out
-        if (this.pager.pending) {
-            logger.debug('pending returning null');
-            return null;
-        }
```

Comment:
This was causing the entire `ResultItem`  - header included - to disappear while data was loading. Removing this didn't seem to cause any issues and ATH passed.